connector/mqtt/vernemq/broker/examples/vernemq.conf

```diff
@@ -5,24 +5,30 @@ listener.http.default = 0.0.0.0:8888
 plugins.vmq_passwd = off
 plugins.vmq_acl = off
 plugins.vmq_k8s.path = /vernemq/plugins/_build/default
-plugins.vmq_k8s = on
+plugins.dojot_acl_plugin.path = /vernemq/dojot_acl_plugin/default
 plugins.dojot_disconnect_plugin.path = /vernemq/dojot_disconnect_plugin/default
+plugins.vmq_k8s = on
+plugins.dojot_acl_plugin = on
 plugins.dojot_disconnect_plugin = on
-plugins.dojot_disconnect_plugin.priority = 2
 plugins.vmq_k8s.priority = 3
-plugins.dojot_acl_plugin.path = /vernemq/dojot_acl_plugin/default
-plugins.dojot_acl_plugin = on
 plugins.dojot_acl_plugin.priority = 2
-leveldb.maximum_memory.percent = 20
+plugins.dojot_disconnect_plugin.priority = 2
 log.console = console
-listener.max_connections = 200000
-listener.nr_of_acceptors = 100
-max_inflight_messages = 20
-max_online_messages = 10000
+log.console.level = info
```

Comment:
just a question, why we don't use error as debug, to avoid too many logs on production?

Reply:
I don't think it will impact that much, and the info log messages can be useful too.